### ISSUE TITLE
Test snapshot packages for v0.10.1

### ISSUE THREAD

**Issue by amarthadan** [api3dao/airnode#1714]
Prior to the v0.10.1 release, the snapshot packages and corresponding Docker images need to be tested.

NPM packages:
```
@api3/airnode-abi@snapshot-v0.10.1
@api3/airnode-adapter@snapshot-v0.10.1
@api3/airnode-admin@snapshot-v0.10.1
@api3/airnode-deployer@snapshot-v0.10.1
@api3/airnode-examples@snapshot-v0.10.1
@api3/airnode-node@snapshot-v0.10.1
@api3/airnode-operation@snapshot-v0.10.1
@api3/airnode-protocol@snapshot-v0.10.1
@api3/airnode-utilities@snapshot-v0.10.1
@api3/airnode-validator@snapshot-v0.10.1
```

Docker images:
```
api3/airnode-deployer-dev:snapshot-v0.10.1
api3/airnode-client-dev:snapshot-v0.10.1
api3/airnode-admin-dev:snapshot-v0.10.1
```

**Comment by dcroote** [comment]
## v0.10.1 testing

tl;dr - everything works, but minor note: there is a warning about the AWS JavaScript SDK version (see end).

**A checked box indicates success.**

Test that npm packages `admin`, `deployer`, and `validator`, which have a `--help` CLI argument, are compiled correctly using `npx`

- [x] admin
  - [x] `generate-mnemonic`
- [x] deployer (tested more below)
- [x] validator
  - [x] Succeeds with `--config "packages/airnode-validator/test/fixtures/config.valid.json" --secrets "packages/airnode-validator/test/fixtures/secrets.valid.env"` (on the latest commit of the v0.10 tag)

The following use the `coingecko-signed-data` integration in `airnode-examples` on Ethereum sepolia, but modified to also test the regular HTTP gateway.

Run Airnode client Docker image locally:

- [x] Testnet request
- [x] HTTP gateway request
- [x] Signed HTTP gateway request
- [x] Withdrawal request

Test AWS deployment with `deployer` Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] `list`
  - [x] `info`
  - [x] `remove`

Test GCP deployment with `deployer` Docker image:

- Requests
  - [x] Testnet request
  - [x] HTTP gateway request
  - [x] Signed HTTP gateway request
  - [x] Withdrawal request
- Deployer commands (with Docker image)
  - [x] `list`
  - [x] `info`
  - [x] `remove-with-receipt` (differs from AWS to achieve additional command coverage)

## Failures

None, but the AWS JavaScript SDK update notification is present in the `deployer` commands and Airnode logs.
This has been fixed in #1659 for v0.11.

```
2023-04-05 23:51:39 Please migrate your code to use AWS SDK for JavaScript (v3).
2023-04-05 23:51:39 For more information, check the migration guide at https://a.co/7PzMCcy
2023-04-05 23:51:39 (Use `node --trace-warnings ...` to show where the warning was created)
2023-04-05 23:52:02 (node:24) NOTE: We are formalizing our plans to enter AWS SDK for JavaScript (v2) into maintenance mode in 2023.
```
